src/Symfony/Component/Validator/Resources/translations/validators.uk.xlf

```diff
@@ -318,6 +318,18 @@
                 <source>Error</source>
                 <target>Помилка</target>
             </trans-unit>
+            <trans-unit id=“83”>
```

Comment:
There's a minor issue with the quotes used in `id="83"`, `id=“84"` and `id=“85”`. You have used "literary quotes" (`“` and `”`) but we must use "programming quotes" (`"`). Thanks!

Reply:
Good catch, fixed in 3b8893fd92